api/external/applications/applications.proto

```diff
@@ -161,7 +161,7 @@ message HabService {
   HealthStatus health_check = 4;
   ServiceStatus status = 5;
 
-  // @afiune Proposal to start receiving informatino about the hab-ring
+  // @afiune Proposal to start receiving information about the hab-ring
   /*repeated string members = 6;*/
```

Comment:
Because of this update you have to recompile the protobuf def. use `compile_go_protobuf_component api` inside the studio